spec.bs

```diff
@@ -1633,6 +1644,7 @@ Several APIs specific to fenced frames are defined on the {{Fence}} interface.
       undefined reportEvent(optional ReportEventType event = {});
       undefined setReportEventDataForAutomaticBeacons(optional FenceEvent event = {});
       sequence&lt;FencedFrameConfig&gt; getNestedConfigs();
+      undefined notifyEvent(optional Event event = {});
```

Comment:
This shouldn't be optional or have a default value, right? The implementation doesn't: https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/core/html/fenced_frame/fence.idl;l=15;drc=d06e0de0964242fff0a093360e9065401d9e0e00

Reply:
Fixed.